src/storageWrappers/LocalForageWrapper.ts

```diff
@@ -26,12 +27,9 @@ export class LocalForageWrapper
   }
 }
 
-interface LocalForageInterface {
+export interface LocalForageInterface<T = string | object | null> {
```

Comment:
I exported `LocalForageInterface` here as it could be useful.

I'm currently writing a wrapper for `LocalForageWrapper` that compresses the cache into a binary, so I need both the generic class and the interface here because I need to use composition over inheritance to be able to turn `LocalForageWrapper` into `PersistentStorage<Uint8Array>`, but make sure the newly created wrapper exposes `string`.